### PR TITLE
PHP-8.4: Document fpow()

### PR DESCRIPTION
**Pull request by Girgias**
~~Waiting on https://github.com/php/php-src/pull/16751~~

### CODE REVIEW

File: reference/math/functions/fpow.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  &reftitle.description;
+  <methodsynopsis>
+   <type>float</type><methodname>fpow</methodname>
+   <methodparam><type>float</type><parameter>num</parameter></methodparam>
+   <methodparam><type>float</type><parameter>exponent</parameter></methodparam>
+  </methodsynopsis>
+  <simpara>
+   Returns the floating point result of raising
+   <parameter>num1</parameter> to the power of <parameter>exponent</parameter>.
+   If <parameter>num1</parameter> is zero and <parameter>exponent</parameter>
```
Comment (cmb69) [inline reference/math/functions/fpow.xml:18]:
```suggestion
   <parameter>num</parameter> to the power of <parameter>exponent</parameter>.
   If <parameter>num</parameter> is zero and <parameter>exponent</parameter>
```

File: reference/math/functions/fpow.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  <variablelist>
+   <varlistentry>
+    <term><parameter>num</parameter></term>
+    <listitem>
+     <simpara>
+      The base to number.
```
Comment (cmb69) [inline reference/math/functions/fpow.xml:30]: Is that proper English? Do you mean "The base number."?

File: reference/math/functions/fpow.xml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <simpara>
+   Returns a <type>float</type> corresponding to
+   <code><parameter>num</parameter><superscript>$exponent</superscript></code>.
```
Comment (cmb69) [inline reference/math/functions/fpow.xml:49]: Hmm, that looks inconsistent. Either both should be `parameter`, or neither. I think the latter makes more sense, so:
```suggestion
   <code>$num<superscript>$exponent</superscript></code>.
```

File: reference/math/functions/fpow.xml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+ </refsect1>
+
+ <refsect1 role="examples">
+  &reftitle.examples;
+  <example xml:id="function.fpow.example.basic">
+   <title><function>fpow</function> example</title>
+   <programlisting role="php">
+<![CDATA[
+<?php
+var_dump(fpow(0, -3));
```
Comment (cmb69) [inline reference/math/functions/fpow.xml:60]: Maybe add a "normal" example (something like `fpow(10, 2)`).

File: reference/math/functions/fpow.xml [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+ </refsect1>
+
+ <refsect1 role="seealso">
+  &reftitle.seealso;
+  <simplelist>
+   <member><link linkend="language.operators.arithmetic"><literal>/</literal></link> - Floating-point division</member>
+   <member><function>pow</function></member>
+   <member><function>fdiv</function></member>
+   <member><function>fmod</function></member>
+   <!-- TODO ** operator link?
```
Comment (cmb69) [inline reference/math/functions/fpow.xml:82]: Maybe. On the other hand, we already link to `pow` what might be good enough.

**Review by cmb69 — COMMENTED** [review]
A couple of nits, but basically looks good. Thank you!

File: reference/math/functions/fpow.xml (hunk 2 above)
Comment (Girgias) [reply reference/math/functions/fpow.xml:30]: Yes I don't know what I did this here

File: reference/math/functions/fpow.xml (hunk 3 above)
Comment (Girgias) [reply reference/math/functions/fpow.xml:49]: Parameter is not allowed within `<superscript>` so that's why I did that, but happy to change.

**Review by cmb69 — APPROVED** [review]
Thanks!

**Pull request merged**